src/components/cards/card.css

```diff
@@ -0,0 +1,171 @@
+:root{
+    --card-badge:rgb(255, 57, 90);
+    --text-overlay:rgb(201, 200, 200);
+    --text-overlay-full: rgb(224, 224, 224);
+}
+.card{
+    position: relative;
+}
+
+.img-xxl{
+    height: 30rem;
+    width: 30rem;
+    object-fit: contain;
+}
+.img-xl{
+    height: 25rem;
+    width: 25rem;
+    object-fit: contain;
+}
+.img-lg{
+    height: 20rem;
+    width: 20rem;
+    object-fit: contain;
+}
+.img-md{
+    height: 15rem;
+    width: 15rem;
+    object-fit: contain;
+}
+.img-sm{
+    height: 12rem;
+    width: 12rem;
+    object-fit: contain;
+
+}
+.img-xsm{
+    height: 5rem;
+    width: 5rem;
+    object-fit: contain;
+
+}
+.card-body{
+    display: flex;
+    flex-direction: column;
+    padding: 10px;
+}
+.card-img{
+    height: 15rem;
+    width: 15rem;
+    object-fit: fill;
+}
+.card-description{
+    font-size: 0.8rem;
+    color: var(--dark-gray-text);
+}
+.card-badge{
+    position: absolute;
+    right: -10px;
+    top: -10px;
+    color: var(--white);
+    width: 120px;
+    height: 120px;
```

Comment:
Use rem unit here for a responsive badge.

Reply:
Convert it to rem

Reply:
Coverted it to rem